README.md

```diff
@@ -36,5 +36,5 @@ Take a look at [our documentation with **installation steps** and **technical in
 [1]: https://github.com/bluerobotics/ping-protocol
 [2]: https://github.com/bluerobotics/ping-viewer/wiki
 [3]: https://github.com/bluerobotics/software-guidelines
-[4]: https://www.bluerobotics.com/store/electronics/ping
+[4]: https://www.bluerobotics.com/store/sensors-sonars-cameras/sonar/ping-sonar-r2-rp/
```

Comment:
We should also update *Device Specifications* to:
https://www.bluerobotics.com/store/sensors-sonars-cameras/sonar/ping-sonar-r2-rp/#tab-technical-details

Reply:
done